packages/styles/scss/utilities/_skeleton.scss

```
@@ -42,3 +42,28 @@
     }
   }
 }
+
+/// Circular Skeleton loading animation
+/// @access public
+/// @example @include circular-skeleton;
+/// @group utilities
+@mixin circular-skeleton {
+  position: relative;
+  overflow: hidden;
+  background: $skeleton-background;
+  border-radius: 50%;
+
+  &::before {
+    position: absolute;
+    width: 200%;
+    height: 100%;
+    animation: 3000ms ease-in-out skeleton-circle infinite;
```

I think using `skeleton` here yields a better result. The animations are out of sync in the deployment:
```suggestion
    animation: 3000ms ease-in-out skeleton infinite;
```
current deployment:

https://github.com/carbon-design-system/carbon/assets/40550942/1c8f38d0-4852-46c6-b01f-30e4cc1ccbf3

using `skeleton`:

https://github.com/carbon-design-system/carbon/assets/40550942/9583768a-a58f-4f31-adb4-bdca65ae9049

if we go this route that also means we can revert the changes in `utilities/_keyframes.scss`

---

Understood! Made the requested change and removed the unwanted keyframe animation from `utilities/_keyframes.scss`. It looks better this way as the animation for the circle and the rectangle are in sync now.